app/lib/server/functions/checkUsernameAvailability.js

```diff
@@ -3,12 +3,12 @@ import s from 'underscore.string';
 
 import { settings } from '../../../settings';
 
-let usernameBlackList = [];
+let usernameBlackList = ['all', 'here'];
 
 const toRegExp = (username) => new RegExp(`^${ s.escapeRegExp(username).trim() }$`, 'i');
 
 settings.get('Accounts_BlockedUsernameList', (key, value) => {
-	usernameBlackList = value.split(',').map(toRegExp);
+	usernameBlackList = usernameBlackList.concat(value.split(',')).map(toRegExp);
```

Comment:
this way you'll never be able to remove something from the blacklist.

for example, if you add something to the setting and save.. if you change again and remove it, the value will not be removed from `usernameBlackList`, because you're always using it's previously value + the value from the setting.

you could do something like this:
```suggestion
	usernameBlackList = ['all', 'here'].concat(value.split(',')).map(toRegExp);
```

Reply:
thank you for your kind guidance ! i commit your changes!